.travis.yml

```diff
@@ -62,7 +60,7 @@ deploy:
   user: jirikuncar
   password:
     secure: QD2ZJ4Oyj4ykvT2uASsLWHUTEHY9cIp0WibRyrTJWFG8uQAPHGFtYNuNTXCP4XOrEFx0OHL6Om+YrsgYT1eR7Aj7GA84TCXcEuPmdQxFdgQuOdvP/jYJyuD1MY5Lf8GdC0m4VdRwm+92d7DCiYSkFwVFzidUtc/eyVdBi3Ay/Iw=
-  distributions: "sdist bdist_wheel"
+  distributions: "compile_catalog sdist bdist_wheel"
```

Comment:
There is no I18N support in `invenio-query-parser`, so I guess `compile_catalog` shouldn't be added...